src/content/docs/en/guides/deploy/netlify.mdx

```diff
@@ -45,18 +45,26 @@ If you prefer to install the adapter manually instead, complete the following tw
       adapter: netlify(),
     });
     ```
- 
-    To render your project using [Netlify's experimental Edge Functions](https://docs.netlify.com/netlify-labs/experimental-features/edge-functions/#app) instead, change the `netlify/functions` import in the Astro config file to use `netlify/edge-functions`.
-      ```js title="astro.config.mjs" ins={3} del={2}
+
+    :::note
+    In version `3.0.0` of `@astrojs/netlify`, the `@astrojs/netlify/functions` package has been enhanced to support Edge middleware directly, eliminating the need for a separate adapter for deploying your entire app to the edge. It's recommended to update your Astro configuration to leverage this enhancement.
+    :::
+
+    Below is the updated configuration snippet. In your `astro.config.mjs`, replace the import statement and update the `adapter` option as shown:
```

Comment:
```suggestion
    You can also deploy your project using Netlify's Edge Functions by adding `edgeMiddleware: true` to the Netlify adatper's configuration:
```
We don't need to refer to what was, since that's what Changelogs are for. This page should document the current state of things, and I think we can make this easier for readers arriving on this page freshand seeing these instructions for the first time by telling them how to enable edge functions, not explaining that you used to do it differently.

Verify the wording above for accuracy, but I think a statement like the one above should be all that's needed here!